examples/tokenvm/cmd/token-cli/cmd/utils.go

```diff
@@ -138,7 +138,7 @@ func promptAmount(
 	return amount, err
 }
 
-func promptInt(
+func promptUint64(
```

Comment:
I recommend keeping both `promptInt` and `promptUint64` (in some cases, we just want an int)